src/pacmod3_node.cpp

```diff
@@ -403,19 +413,23 @@ int main(int argc, char *argv[])
   ros::Subscriber shift_cmd_sub = n.subscribe("as_rx/shift_cmd", 20, callback_shift_set_cmd);
   ros::Subscriber steer_cmd_sub = n.subscribe("as_rx/steer_cmd", 20, callback_steer_cmd_sub);
   ros::Subscriber turn_cmd_sub = n.subscribe("as_rx/turn_cmd", 20, callback_turn_signal_set_cmd);
+  ros::Subscriber rear_pass_door_cmd_sub = n.subscribe("as_rx/rear_pass_door_cmd", 20, callback_rear_pass_door_set_cmd);
 
   // Populate rx list
   std::shared_ptr<LockedData> accel_data(new LockedData);
   std::shared_ptr<LockedData> brake_data(new LockedData);
   std::shared_ptr<LockedData> shift_data(new LockedData);
   std::shared_ptr<LockedData> steer_data(new LockedData);
   std::shared_ptr<LockedData> turn_data(new LockedData);
+  std::shared_ptr<LockedData> door_data(new LockedData);
```

Comment:
Looks like you've got a leftover here from the previous name.

Reply:
No, I haven't added this line. This is actually for the door report that we had previously.

Reply:
well, I take that back. You are right!